cmd/nerdctl/stats_linux_test.go

```diff
@@ -32,9 +33,12 @@ func TestStats(t *testing.T) {
 		t.Skip("test skipped for rootless containers on cgroup v1")
 	}
 	testContainerName := testutil.Identifier(t)
+	exitedTestContainerName := fmt.Sprintf("%s-exited", testContainerName)
 
 	base := testutil.NewBase(t)
 	defer base.Cmd("rm", "-f", testContainerName).Run()
+	defer base.Cmd("rm", "-f", exitedTestContainerName).Run()
+	base.Cmd("run", "--name", exitedTestContainerName, testutil.AlpineImage, "echo", "'exited'").AssertOK()
```

Comment:
can you plz check that  `nerdctl stats` is a generating  some output when a container exist ?